Paging Data Source Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -1,1453 +1,799 @@
-		<dict>
-			<key>buildActionMask</key>
-			<string>2147483647</string>
-			<key>files</key>
-			<array>
-				<string>C56F4C68708954728D408790</string>
-				<string>58CB0D68E7C31995DFDC1C78</string>
-			</array>
-			<key>isa</key>
-			<string>PBXFrameworksBuildPhase</string>
-			<key>runOnlyForDeploymentPostprocessing</key>
-			<string>0</string>
-		</dict>
-		<key>F040DA954D197874E5D43E07</key>
-		<dict>
-			<key>includeInIndex</key>
-			<string>1</string>
-			<key>isa</key>
-			<string>PBXFileReference</string>
-			<key>lastKnownFileType</key>
-			<string>sourcecode.c.objc</string>
-			<key>name</key>
-			<string>VOKPagingFetchedResultsDataSource.m</string>
-			<key>path</key>
-			<string>Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m</string>
-			<key>sourceTree</key>
-			<string>&lt;group&gt;</string>
-		</dict>
-		<key>F1574E970AFE9B5749CDA1CB</key>
-		<dict>
-			<key>fileRef</key>
-			<string>686782311C358BE339DC1BF4</string>
-			<key>isa</key>
-			<string>PBXBuildFile</string>
-			<key>settings</key>
-			<dict>
-				<key>COMPILER_FLAGS</key>
-				<string>-fobjc-arc</string>
-			</dict>
-		</dict>
-		<key>F20406F4F8B4C8E6329C6CC9</key>
-		<dict>
-			<key>explicitFileType</key>
-			<string>archive.ar</string>
-			<key>includeInIndex</key>
-			<string>0</string>
-			<key>isa</key>
-			<string>PBXFileReference</string>
-			<key>path</key>
-			<string>libPods-Paging Data Source Example.a</string>
-			<key>sourceTree</key>
-			<string>BUILT_PRODUCTS_DIR</string>
-		</dict>
-		<key>F98ED746F2684F11FFA7DCB1</key>
-		<dict>
-			<key>includeInIndex</key>
-			<string>1</string>
-			<key>isa</key>
-			<string>PBXFileReference</string>
-			<key>lastKnownFileType</key>
-			<string>sourcecode.c.h</string>
-			<key>name</key>
-			<string>VOKCoreDataManagerInternalMacros.h</string>
-			<key>path</key>
-			<string>Pod/Classes/Internal/VOKCoreDataManagerInternalMacros.h</string>
-			<key>sourceTree</key>
-			<string>&lt;group&gt;</string>
-		</dict>
-		<key>F9EE6C7D08EAE47FEF1469FE</key>
-		<dict>
-			<key>includeInIndex</key>
-			<string>1</string>
-			<key>isa</key>
-			<string>PBXFileReference</string>
-			<key>lastKnownFileType</key>
-			<string>text.xcconfig</string>
-			<key>path</key>
-			<string>Pods-Paging Data Source Example-Vokoder.xcconfig</string>
-			<key>sourceTree</key>
-			<string>&lt;group&gt;</string>
-		</dict>
-		<key>FB475FCA5A8D1A46ABA6681F</key>
-		<dict>
-			<key>includeInIndex</key>
-			<string>1</string>
-			<key>isa</key>
-			<string>PBXFileReference</string>
-			<key>lastKnownFileType</key>
-			<string>sourcecode.c.h</string>
-			<key>name</key>
-			<string>VOKFetchedResultsDataSource.h</string>
-			<key>path</key>
-			<string>Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h</string>
-			<key>sourceTree</key>
-			<string>&lt;group&gt;</string>
-		</dict>
-	</dict>
-	<key>rootObject</key>
-	<string>32AD798924C101FC1DBC1D63</string>
-</dict>
-</plist>
+// !$*UTF8*$!
```

Comment:
What version of CocoaPods are you running? I haven't seen the wholesale rewrite of the plist problem with .37.2